connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorConnectorConfig.java

```diff
@@ -169,6 +170,7 @@ static Map<String, Object> sourceConsumerConfig(Map<String, ?> props) {
         result.putAll(Utils.entriesWithPrefix(props, CONSUMER_CLIENT_PREFIX));
         result.putAll(Utils.entriesWithPrefix(props, SOURCE_PREFIX + CONSUMER_CLIENT_PREFIX));
         result.put(ENABLE_AUTO_COMMIT_CONFIG, "false");
+        result.put(ALLOW_AUTO_CREATE_TOPICS_CONFIG, "false");
```

Comment:
While this may be more ideal for many use cases, it's a potentially-breaking change that I don't believe we can make without waiting for a major release (right now, that'd be 4.0.0), and an accepted KIP. If you're unfamiliar with the KIP process, you can read more about it here: https://cwiki.apache.org/confluence/display/KAFKA/Kafka+Improvement+Proposals#KafkaImprovementProposals-Purpose

Reply:
@C0urante OK, I will draft a KIP later.

Reply:
@aaron-ai thanks for drafting a KIP that quickly.  I left a couple of comments there.